### PR TITLE
[DataFactory]Add sensitive fields extracted from connection string in swagger for AKV reference

### PR DESCRIPTION
**Pull request by boxiao-o**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Can you please update the affected examples with the new fields?

**Comment by boxiao-o** [comment]
@jhendrixMSFT May I know which file to update? If `LinkedServices_Create.json` in `examples` folder, those fields I added is optional and will only be used in some special cases, so may not need to be added there.

**Comment by jhendrixMSFT** [comment]
@boxiao-o If these fields are optional then perhaps it makes more sense to add a new example that uses them?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -3763,6 +3831,10 @@
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
           "type": "object"
         },
+        "pwd": {
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Why is this one named pwd while others password?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 2]
```diff
@@ -2222,6 +2270,10 @@
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
           "type": "object"
         },
+        "credString": {
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Here too, please use full words in names

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 1 above)
Comment (boxiao-o) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: The property name needs to be same with the name in connection string. In this Netezza linked service case, a typical connection string is `Server=<server>;Port=<port>;Database=<database>;UID=<user name>;PWD=<password>`.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 2 above)
Comment (boxiao-o) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Same here. This field will be back-filled into connection string, so the name should be the same.

**Comment by boxiao-o** [comment]
@jhendrixMSFT Seems based on our convention, we usually do not update example here, but I will update the examples in sdk [JsonSamples](https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/DataFactory/DataFactory.Tests/JsonSamples)

**Comment by jhendrixMSFT** [comment]
@boxiao-o I'm not sure what you mean by "based on our convention".  The example files are used to generate documentation so if you want these fields doc'ed you should add an example for them.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**